### DISCUSSION TITLE
Is Nuxt currently suitable for the project?and are there many pitfalls（Nuxt现在适合做项目吗？坑多吗）

### DISCUSSION THREAD

**Discussion by lixiaofa** [nuxt/nuxt#30240] [Questions]
Is Nuxt suitable for projects now, and are there many pitfalls（Nuxt现在适合做项目吗？坑多吗）

For example, in a shopping mall project, I mean an integrated database and backend services（比如商城项目， 一把梭哈， 我指的是集成数据库，后端服务）

**Comment by xjccc** [comment]
Nuxt3 is very powerfull. And has lots of modules and examples to help devlopers make web site. And the maintainer deal with problems very quickly, don't worry, let's get it.

**Reply by lixiaofa** [reply]
I heard that the backend is not easy to use, such as integrating databases, orm

It is rare to use nuxt for projects on GitHub

**Comment by xjccc** [comment]
Its easy to use with `useStorage`.

[server-storage](https://nuxt.com/docs/guide/directory-structure/server#server-storage)

**Reply by lixiaofa** [reply]
Do you have any good project recommendations, more complex ones that integrate databases

**Reply by xjccc** [reply]
Emm.. I have no idea about this. Every scene is not similar. And try it do best.

**Reply by lixiaofa** [reply]
Recommend a good project using nuxt ，

**Reply by xjccc** [reply]
You could search `nuxt3` on github. And there are so many good projects. You could try it.

[learn nuxt](https://github.com/nuxt/learn.nuxt.com)
